go_api/infrastructure/event.go

```diff
@@ -0,0 +1,58 @@
+// DB操作
+package infrastructure
+
+import (
+	"github.com/NUTFes/lottery/go_api/domain"
+	"gorm.io/gorm"
+)
+
+type EventInfrastructure struct {
+	db *gorm.DB
+}
+
+func NewEventInfrastructure(db *gorm.DB) *EventInfrastructure {
+	return &EventInfrastructure{db: db}
+}
+
+// 全イベントの取得
+func (u *EventInfrastructure) FindAll() (*domain.Events, error) {
+	events := domain.Events{}
+	if err := u.db.Find(&events).Error; err != nil {
+		return nil, err
+	}
+	return &events, nil
+}
```

Comment:
```suggestion
func (e *EventInfrastructure) FindAll() (*domain.Events, error) {
	events := domain.Events{}
	if err := e.db.Find(&events).Error; err != nil {
		return nil, err
	}
	return &events, nil
}
```
レシーバーの値が`u`のままなので`e`に変更お願いします。